[2024-12-07T20:07:21.525+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-07T20:07:21.544+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-06T00:00:00+00:00 [queued]>
[2024-12-07T20:07:21.556+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-06T00:00:00+00:00 [queued]>
[2024-12-07T20:07:21.558+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-07T20:07:21.577+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-06 00:00:00+00:00
[2024-12-07T20:07:21.589+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=158) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-07T20:07:21.592+0000] {standard_task_runner.py:72} INFO - Started process 174 to run task
[2024-12-07T20:07:21.592+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb', 'load_data', 'scheduled__2024-12-06T00:00:00+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb.py', '--cfg-path', '/tmp/tmp2fe784zo']
[2024-12-07T20:07:21.594+0000] {standard_task_runner.py:105} INFO - Job 214: Subtask load_data
[2024-12-07T20:07:21.653+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-06T00:00:00+00:00 [running]> on host 8553af9ff67b
[2024-12-07T20:07:22.007+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-06T00:00:00+00:00'
[2024-12-07T20:07:22.009+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-07T20:07:22.009+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-07T20:07:22.010+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-12-07 20:07:21.545855+00:00
[2024-12-07T20:07:22.010+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb and current dag run status:running
[2024-12-07T20:07:22.010+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T20:10:16.005+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T20:10:16.032+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-06T00:00:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2024-12-09T20:10:16.043+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: etl_data_pipeline_mongodb.load_data scheduled__2024-12-06T00:00:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2024-12-09T20:10:16.049+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
